static/templates/user_profile_modal.hbs

```diff
@@ -64,16 +64,22 @@
                 </div>
             </div>
 
-            <div class="subscription-group-list">
-                <div class="tabcontent" id="groups-tab">
-                    <table class="user-group-list" data-empty="{{t 'No user group subscriptions.'}}"></table>
+            <div class="subscription-stream-tab">
+                <div class="subscription-stream-list">
+                    <div class="tabcontent" id="streams-tab">
```

Comment:
This level of nesting is not needed here. I think we can achieve the same effect by removing the last div here and putting its classes and id to the first div.

Reply:
Yeah, that seems likely.